app/controllers/topics.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+helpers do
+  def res(status_code, message)
+    status status_code
+    {
+      success: true,
+      status: status_code,
+      message: message
+    }.to_json
+  end
+end
+
+get '/topics' do
+  require_admin
+  @topics = Topic.all
+  gon.topic_titles = @topics.map(&:title)
+  @user = User.first(email: session[:email])
+  gon.user_topics = @user.topics.map(&:title) unless @user.topics.empty?
+  @digests = @user.userdigests
```

Comment:
`userdigests` here seems to be miswritten. If you want to use two words for a method, separate them with an underscore

scondly `@user.userdigests` sounds like a repeatition. If the method `userdigests` is in the user scope, there is no need to have the user prefix. `digests` will just be fine